src/module/actor/base.ts

```diff
@@ -460,13 +460,16 @@ class ActorPF2e extends Actor<TokenDocumentPF2e, ItemTypeMap> {
     }
 
     getModifierAdjustments(selectors: string[], slug: string): ModifierAdjustment[] {
-        return Array.from(
+        const adjustmentsRecord = this.synthetics.modifierAdjustments;
+        const selectorMatches = Array.from(
             new Set(
-                selectors
-                    .flatMap((s) => this.synthetics.modifierAdjustments[s] ?? [])
-                    .filter((a) => [slug, null].includes(a.slug))
+                selectors.includes("all")
+                    ? Object.values(adjustmentsRecord).flat()
+                    : selectors.flatMap((s) => this.synthetics.modifierAdjustments[s] ?? [])
```

Comment:
```suggestion
                    : selectors.flatMap((s) => adjustmentsRecord[s] ?? [])
```

if you're going through the trouble of aliasing, might as well use it?